composer.json

```diff
@@ -33,17 +33,25 @@
             "role": "Maintainer"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/kienngd/apiato_core.git"
```

Comment:
WARN

Fork, not a link to the official repository

Reply:
Because of apiato core need to upgrade laravel 11 too. I fork it, upgrade and make pull request for apiato core.